app/views/ems_infra_dashboard/show.html.haml

```diff
@@ -0,0 +1 @@
+= render :partial => "shared/views/show_containers_dashboard"
```

Comment:
Why do we need partials that just render other partials?

And why do we need a non-partial view that just calls a partial. That is 2 extra files :-(

That's not a new problem here, it's in the older code too:
```
devil - [~/Projects/manageiq] (master)$ grep -r show_containers_dashboard app/
app/views/container_dashboard/show.html.haml:= render :partial => "shared/views/show_containers_dashboard"
app/views/ems_container/_show_dashboard.html.haml:= render :partial => "shared/views/show_containers_dashboard"
```